deployment/nic-configuration-operator-chart/templates/config-daemon.yaml

```diff
@@ -24,6 +24,7 @@ spec:
       nodeSelector: {{- toYaml .Values.operator.nodeSelector | nindent 8 }}
       serviceAccountName: {{ include "nic-configuration-operator.serviceAccountName" . }}
       terminationGracePeriodSeconds: 10
+      hostNetwork: true
```

Comment:
why do we need host network?

Reply:
We use netlink socket to apply RoCE settings and we need the host network for that